### PR TITLE
Fix EsFileThumbnail Image Stretching

### PR DESCRIPTION
**Pull request by gigiwargin**
### 🔗 Linked issue
https://energysage.atlassian.net/browse/CSM-2856

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [X] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Updated `EsFileThumbnail` to stop stretching images. Most images of standard letter dimensions (8.5" x 11") will appear fine. Extremely wide and short images will also be fine. The outlier is an extremely narrow and tall image, but I expect the use case for this to be small.

### 🥼 Testing

Manual 

#### 🧐 Feedback Requested / Focus Areas

- Do you have concerns about the files that will show a thumbnail with a column of white space (narrow and tall images)?

### 📝 Checklist

- [X] I have linked an issue or discussion.
- [X] I have updated the documentation accordingly.
- [X] I have documented testing approach

### CODE REVIEW

File: es-vue-base/src/lib-components/EsFileThumbnail.vue [hunk 1]
```diff
@@ -99,9 +99,8 @@
                             'thumbnail-inner-wrapper': loading,
                         }">
                         <template v-if="!loading">
-                            <div class="h-100 w-100">
+                            <div class="h-100">
```
Comment (satwikgoyal) [inline es-vue-base/src/lib-components/EsFileThumbnail.vue:102]:
```suggestion
                            <div class="image-preview-div">
```

File: es-vue-base/src/lib-components/EsFileThumbnail.vue [hunk 2]
```diff
@@ -339,13 +338,8 @@ a:hover {
...
 .image-preview {
-    @include media-breakpoint-up(md) {
-        min-height: 100px;
-    }
-
-    @include media-breakpoint-up(lg) {
-        min-height: 150px;
-    }
+        float: left;
+    height: 200%;
```
Comment (satwikgoyal) [inline es-vue-base/src/lib-components/EsFileThumbnail.vue:342]:
```suggestion
    height: 100%;
    object-fit: cover;
    width: 100%;
```

File: es-vue-base/src/lib-components/EsFileThumbnail.vue [hunk 3]
```diff
@@ -339,13 +338,8 @@ a:hover {
 }
```
Comment (satwikgoyal) [inline es-vue-base/src/lib-components/EsFileThumbnail.vue:339]:
```suggestion
.image-preview-div {
    height: 100%;
    overflow: hidden;
    width: 100%;
}
```

**Review by satwikgoyal — CHANGES_REQUESTED** [review]
@gigiwargin these changes look great! 🪄 I tried some CSS which seem to cover the edge case(s) too, please let me know what are your thoughts on it. Currently it is using the center of the image but we can add the `float` and `vertical-align` css properties if we want to show the top-left corner as the thumbnail cover.

**Comment by gigiwargin** [comment]
Ty for the suggestions!!

File: es-vue-base/src/lib-components/EsFileThumbnail.vue [hunk 4]
```diff
@@ -99,10 +99,9 @@
                             'thumbnail-inner-wrapper': loading,
                         }">
                         <template v-if="!loading">
-                            <div class="h-100 w-100">
+                            <div class="image-preview-div h-100 w-100 overflow-hidden">
```
Comment (satwikgoyal) [inline es-vue-base/src/lib-components/EsFileThumbnail.vue:102]:
```suggestion
                            <div class="h-100 w-100 overflow-hidden">
```

**Review by satwikgoyal — APPROVED** [review]
Looks great @gigiwargin. Since you were able to use inline styles we can remove the `image-preview-div` class and then we are all set to go.

**Pull request merged**